### PR TITLE
fix import string, change name for supply_domain to domain

### PR DESCRIPTION
**Pull request by s3341458**

### CODE REVIEW

**Comment by s3341458** [comment]
I found a bug in my serious backref, when I am working on the testcase,
need another reivew @DanielCollins

**Comment by DanielCollins** [comment]
there's a bunch of weird stuff here. why do you need to change the version of lxml to change the supply domain model?

File: python/requirements/dev.txt [hunk 1]
```diff
@@ -30,5 +30,5 @@ beautifulsoup4==4.6.0
 inflection==0.3.1
 pyOpenSSL==17.5.0
 pytest==3.6.1
-lxml==3.8.0
+lxml==4.2.5
```
Comment (DanielCollins) [inline python/requirements/dev.txt]: this really needs to be kept in sync with the version in the other repository

**Comment by s3341458** [comment]
I need to write a testcase ensure supply_domain backrefs are all there

**Comment by s3341458** [comment]
end up notice backref with dynamic import works fine the problem is due to another thing.

File: python/requirements/dev.txt (hunk 1 above)
Comment (DanielCollins) [reply python/requirements/dev.txt]: more to the point, why do you need this?

Comment (s3341458) [reply python/requirements/dev.txt]: I will remove this

**Comment by s3341458** [comment]
@DanielCollins  I have bring back lxml to old version

**Comment by DanielCollins** [comment]
raises the test coverage a little bit 👍

**Pull request merged**